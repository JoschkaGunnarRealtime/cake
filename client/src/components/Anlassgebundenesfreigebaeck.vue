<template>
    <div>
      <v-progress-circular indeterminate v-if="isLoading"/>
      <div v-else>
        <h1>Hallo ich bin ein Anlass gebundenes FreigebÃ¤ck!</h1>
        <div
            v-for="(anlassgebundenesfreigebaeckbedarfsanteil, index) in anlassgebundenesfreigebaeckbedarfsanteile"
            :key="index"
            class="d-inline-flex">
          <Anlassgebundenesfreigebaeckbedarfsanteil
            v-if="anlassgebundenesfreigebaeckbedarfsanteil.type.includes('anlassgebundenesfreigebaeckbedarfsanteil')"
            :anlassgebundenesfreigebaeckbedarfsanteilsidentifikationsnummer="anlassgebundenesfreigebaeckbedarfsanteil._id"
            :bedienungsanleitung="anlassgebundenesfreigebaeckbedarfsanteil.content.bedienungsanleitung"
            :url="anlassgebundenesfreigebaeckbedarfsanteil.content.url"/>
          <Kommentarzettel 
            v-if="anlassgebundenesfreigebaeckbedarfsanteil.type.includes('zettel')"
            :kommentarzettel="'oh noes!'" />
        </div>
      </div>
    </div>
</template>

<script>
import Anlassgebundenesfreigebaeckbedarfsanteil from './Anlassgebundenesfreigebaeckbedarfsanteil';
import Kommentarzettel from './Kommentarzettel.vue';

export default {
  name: 'Anlassgebundenesfreigebaeck',
  components: {
    Anlassgebundenesfreigebaeckbedarfsanteil,
    Kommentarzettel,
  },
  data() {
    return {
      anlassgebundenesfreigebaeckbedarfsanteile: [],
      isLoading: true,
    };
  },
  beforeMount() {
    this.axios.get('/anlassgebundenesfreigebaeck/1').then((response) => {
      console.log(response, 'Nudelsuppe');
      this.anlassgebundenesfreigebaeckbedarfsanteile = response.data.anlassgebundenesfreigebaeckbedarfsanteile;
      this.isLoading = false;
    });
  },
};
</script>

<style lang="scss" scoped>
  .kuchentrennschneider{ 
    @extend .d-flex !optional; 
  } 
</style>