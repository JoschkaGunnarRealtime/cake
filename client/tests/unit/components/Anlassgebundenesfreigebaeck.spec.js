import Anlassgebundenesfreigebaeck from '../../../src/components/Anlassgebundenesfreigebaeck.vue';
import { mount } from '@vue/test-utils';

describe('Anlassgebundenesfreigebaeck', () => {
    it('should take and compare a snapshot', () => {
        const wrapper = mount(Anlassgebundenesfreigebaeck);
        expect(wrapper).toMatchSnapshot();
    });
});