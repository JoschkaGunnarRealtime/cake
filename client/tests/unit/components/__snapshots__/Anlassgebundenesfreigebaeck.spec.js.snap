// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Anlassgebundenesfreigebaeck should take and compare a snapshot 1`] = `
<div>
  <v-progress-circular indeterminate=""></v-progress-circular>
</div>
`;
